experiments/yarn/src/main/scala/org/apache/gearpump/experiments/yarn/master/AmActor.scala

```diff
@@ -312,7 +310,7 @@ object YarnApplicationMaster extends App with ArgumentsParser {
    * For yet unknown reason this is needed for my local pseudo distributed cluster.   
    * 
    */
-  def getForcedDefaultYarnConf:Configuration = {
+  def getForcedDefaultYarnConf:YarnConfiguration = {
```

Comment:
```scala
 object YarnApplicationMaster extends App with ArgumentsParser {
```
Can you document what is YarnApplicationMaster? 
Should we move YarnApplicationMaster to a seperate file instead declaring in file AmActor?